css/app.css

```diff
@@ -683,7 +683,7 @@ a:hover .icon.out-link   { background-position: -500px -14px;}
     float: left;
     height: 100%;
     overflow: hidden;
-    z-index: 2;
+    z-index: 10;
```

Comment:
This change messes up z-index order:
![screen shot 2015-03-15 at 10 17 48 pm](https://cloud.githubusercontent.com/assets/108094/6659675/25314f9c-cb61-11e4-86cc-6c57bece3560.png)